bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+suhosin.executor.func.blacklist="escapeshellarg, escapeshellcmd, exec, highlight_file, lchgrp, lchown, link, symlink, passthru, pclose, popen, proc_close, prog_get_status, proc_nice, proc_open, proc_terminate, shell_exec, show_source, system, gc_collect_cycles, gc_enable, gc_disable, gc_enabled, getmypid, getmyuid, getmygid, getrusage, getmyinode, get_current_user, libxml_disable_entity_loader, parse_str, phpinfo, phpversion"
```

Comment:
Because the client library is using `php_uname()` which is disabled as a dangerous function. I agree that the current way of enabling this again is terrible though.

Re: `nginx.conf`
Having custom web server config for php apps is very common and I don't see it's an issue.